dashboard/apps/dashboard/management/commands/sync.py

```diff
@@ -111,6 +112,28 @@ def sync_clients(data_dir):
             client.save()
 
 
+def sync_departments(data_dir):
+    logging.info('sync departments')
+    source = os.path.join(data_dir, 'departments.json')
+    with open(source, 'r') as sf:
+        data = json.loads(sf.read())
+    for item in data:
+        useful_data = {
+            'float_id': item['department_id'],
+            'name': item['department_name'],
+            'raw_data': item,
+        }
```

Comment:
The `compare` function could be converted to a method in a `Comparable` mixin class that the model classes inherit from, for example